workshop1/03-connecting-the-api.md

```diff
@@ -135,6 +136,12 @@ as application state, and is fetched when the page first loads.
 3. Update the template returned by your app component to reference the `pets`
    state variable instead of `mockedPets`.
 
+4. Navigate to the node terminal, within Visual Studio Code. You will notice a yellow message saying your code was `Compiled with warnings`. Because we've changes a few things during this step, there are some variables we are not using anymore: `logo` and `mockedPets`.  
+Even though they don't have any effect on the code, it's good practice to keep your code as clean as possible, and it includes making sure you don't have any unused code (which will clutter your project as it accumulates, and don't bring any value).  
+_Always keep a look at this section of your project: ideally we want our projects to be warning-free (even though sometimes that's not possible)._
+
+5. After removing the unused variables, `node` now should be displaying a green message saying "Compiled succesfully".
```

Comment:
nitpick: this is a result of step 4, so I wouldn't use another step for it. 

Also, other nitpick: that message I guess comes from `npm start` which launches some script packaged in `react-scripts` that launches the Webpack development server, which in turn triggers the transpilation from Typescript upon detecting changes on the file system etc, you got the idea - so `node` sounds a bit too generic in this context.. I'd use simply "the console should now..."

Reply:
Also: `s/succesfully/successfully/` :)

Reply:
Yeah, that makes sense, thanks!